DynamoDB/StandaloneFSharp/Program.fs

```diff
@@ -0,0 +1,64 @@
+﻿open Amazon.DynamoDBv2
+open Amazon.DynamoDBv2.DocumentModel
+open Amazon.DynamoDBv2.Model
+open Amazon.Runtime
+open RealtyShares
+open RealtyShares.DynamoDb
+open RealtyShares.DynamoEventStore
+
+[<EntryPoint>]
+let main argv = 
+    // Use credentials from environmental variables, and "N. California",
+    // which is `Amazon.RegionEndpoint.USWest1`, is our default. -- D.K.
+    let credentials = new EnvironmentVariablesAWSCredentials()
+    let client = new AmazonDynamoDBClient(credentials, Amazon.RegionEndpoint.USWest1)
+    let tableName = "StandaloneFSharp-demo-table"
+
+    async {
+        printfn "Creating table, %s..." tableName
+        let! res = createTable client tableName
+        let! status = waitForStatus client tableName TableStatus.ACTIVE
+        printfn "Table created!"
+
+        let table = Table.LoadTable(client, tableName)
+
+        let sw = new System.Diagnostics.Stopwatch()
+        sw.Start()
+    
+        printfn "inserting %d entries..." 1000
+        try 
+            async {
+                let inserts = 
+                    [0..1000]
```

Comment:
`[1..1000]` or `[0..999]`, or, better yet, use `let N = 1000` and then `[1..N]`.